content/numpy/concepts/math-methods/terms/hypot/hypot.md

```diff
@@ -0,0 +1,90 @@
+---
+Title: '.hypot()'
+Description: 'Returns the hypotenuse of a right triangle, given the "legs".'

```

Comment:
According to the [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md), we should not use apostrophes or quotes in Title: or Description:.

```suggestion
Description: 'Returns the hypotenuse of a right triangle, given the legs.'
```